packages/@react-aria/utils/src/inertValue.ts

```diff
@@ -0,0 +1,11 @@
+import {version} from 'react';
+
+export function inertValue(value: any) {
+  const pieces = version.split('.');
+  const major = parseInt(pieces[0], 10);
+  if (major >= 19) {
+    return !!value;
```

Comment:
once the arg is typed correctly, this will make more sense i think
```suggestion
    return value ?? undefined;
```